src/components/stepforms/FilterStepForm.vue

```diff
@@ -65,6 +70,24 @@ export default class FilterStepForm extends BaseStepForm<FilterStep> {
   set conditions(newval) {
     this.editedStep.condition.and = [...newval];
   }
+
+  submit() {
+    for (const cond of this.editedStep.condition.and as FilterSimpleCondition[]) {
+      const type = this.columnHeaders.filter(h => h.name === cond.column)[0].type;
```

Comment:
This is getting a more and more common operation. Shouldn't we have a helper somewhere or a getter on the store?